src/backend/launcher.ts

```diff
@@ -121,16 +124,32 @@ async function prepareLaunch(
     gameSettings.useSteamRuntime &&
     (isNative || gameSettings.wineVersion.type === 'proton')
   if (shouldUseRuntime) {
+    let nonNativeRuntime: SteamRuntime['type'] = 'soldier'
+    if (!isNative) {
+      try {
+        const parentPath = normalize(join(gameSettings.wineVersion.bin, '..'))
+        const requiredAppId = VDF.parse(
+          readFileSync(join(parentPath, 'toolmanifest.vdf'), 'utf-8')
+        )['manifest']['require_tool_appid']
```

Comment:
Why dont just use dot syntax here
```suggestion
        ).manifest?.require_tool_appid
```